eth/ethconfig/config.go

```diff
@@ -111,6 +111,7 @@ var Defaults = Config{
 	RPCTxFeeCap: 1, // 1 ether
 
 	ArbRPCEVMTimeout: 5 * time.Second,
+	L2RPCAddr:        "http://localhost:8547",
```

Comment:
maybe not set this, since it will be reset to empty or defined by flag in https://github.com/erigontech/erigon/blob/835bb523b8835672eab6642ed126395373eba490/turbo/cli/flags.go#L79-L83

or change default value in flag to the same value

Reply:
const was there while flag prasing was not working, now removed